Tests/Thumbnail/ConsumerThumbnailTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Sonata\MediaBundle\Tests\Thumbnail;
+
+use Sonata\MediaBundle\Thumbnail\ConsumerThumbnail;
+use Sonata\MediaBundle\Thumbnail\ThumbnailInterface;
+use Sonata\NotificationBundle\Backend\BackendInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Sonata\MediaBundle\Provider\MediaProviderInterface;
+use Sonata\MediaBundle\Model\MediaInterface;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class ConsumerThumbnailTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGenerateDispatchesEvents()
+    {
+        $thumbnail = $this->createMock(ThumbnailInterface::class);
```

Comment:
`createMock` doesn't work with old versions of php that we are supporting… so does the `::class` notation. Please replace them with `getMock` (there will be a warning, but it should pass), and the classname as a string